extensions/control-plane/api/management-api/asset-api/src/test/java/org/eclipse/edc/connector/api/management/asset/AssetApiControllerTest.java

```diff
@@ -54,17 +54,12 @@
 import static org.mockito.Mockito.when;
 
 @ApiTest
-public class AssetApiControllerIntegrationTest extends RestControllerTestBase {
+public class AssetApiControllerTest extends RestControllerTestBase {
```

Comment:
I agree on remove the `Integration` word, as the test type is already described by the annotation (Api).